packages/fxa-settings/src/components/Settings/Page2faReplaceRecoveryCodes/index.tsx

```diff
@@ -174,15 +174,19 @@ export const Page2faReplaceRecoveryCodes = (_: RouteComponentProps) => {
                 type="button"
                 className="cta-neutral cta-base-p mx-2 flex-1"
                 onClick={goHome}
```

Comment:
I noticed the Glean event isn't emitted with this onClick handler, but changing it to the following suggestion seems to fix it. (Should also work for the other cancel buttons)
```suggestion
               onClick={() => {
                  goHome();
                }}
```